operations/template/docs.tf

```diff
@@ -27,7 +27,22 @@ resource "azurerm_storage_account" "docs" {
       tags["support_group"],
       tags["system"],
       tags["technical_steward"],
-      tags["zone"]
+      tags["zone"],
+      customer_managed_key,
     ]
   }
+
+  identity {
+    type = "SystemAssigned"
+  }
+}
+
+
+resource "azurerm_key_vault_managed_storage_account" "key_vault_storage_account" {
```

Comment:
it looks like this resource isn't referenced by anything else - is that expected?

Reply:
Yes, this is expected.  There can be plenty of "leaf" nodes (or resources) that aren't referenced.